client/jetpack-connect/authorize.js

```diff
@@ -141,6 +142,15 @@ export class JetpackAuthorize extends Component {
 		isRedirecting: false,
 	};
 
+	componentDidMount() {
+		// If we have a logged-in user
+		// Preload the setup image that will be used when the user clicks on the connect botton.
+		// This is needed to avoid text jumping when the image is loaded.
+		if ( this.props.user ) {
```

Comment:
Thanks @chihsuan 

I might have tested it incorrectly in that case. I'll look into it again.

> Since the image is only for Woo, should we check if the user is on a Woo JPC flow before preloading the image?

Good catch. I'll make an update.

Reply:
@chihsuan I've converted svg image to React components in [Convert svg to react component](https://github.com/Automattic/wp-calypso/pull/96558/commits/e1522e4a4aab470a03dea113c4d34ae10a327860)